announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/DbAnnouncementService.java

```diff
@@ -632,6 +633,8 @@ public Map<String, List<AnnouncementMessage>> getViewableAnnouncementsForCurrent
 
 	public List<AnnouncementMessage> getViewableAnnouncementsForSite(String channelRef, Integer maxAgeInDays) {
 
+		List<AnnouncementMessage> viewableAnnouncements = new Vector();
```

Comment:
@austin48  a Vector probably isn't needed here unless it would be used by another thread?

Reply:
Although the main case for this commit is returning an empty List where there is a failure getting messages, I used a Vector because in a successful case, getMessages() (from BaseAnnouncementService) returns a Vector.

Reply:
Minor thing, but if the variable is not going to be used elsewhere I would directly return "new Vector" at the end of the method.